### PR TITLE
UI: VAULT-6519 PKI configuration read 

### PR DESCRIPTION
**Pull request by kiannaquach**
**Description**
- Instead of creating a model for `pki/config`, we opted to create separate models for each section
- For the configuration page, we only show CRL (`pki/crl` model), URL (`pki/urls` model), and mount config (`secret-engine` which belongs to the `mount-config` model).

Screenshot 2023-03-21 at 3 33 57 PM

### CODE REVIEW

File: ui/app/adapters/pki/crl.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+export default class PkiCrlAdapter extends ApplicationAdapter {
+  namespace = 'v1';
+
+  _url(backend) {
+    return `${this.buildURL()}/${encodePath(backend)}/config/crl`;
+  }
+
+  findRecord(store, type, id) {
+    return this.ajax(this._url(id), 'GET').then((resp) => {
```
Comment (hashishaw) [inline ui/app/adapters/pki/crl.js:17]: would it be too much trouble to change the `id` param here to `backend` so it's more clear what it's expecting? ID could be any number of things, and it's usually not the backend path 😄

File: ui/tests/acceptance/pki/pki-engine-workflow-test.js [hunk 2]
```diff
@@ -73,15 +73,6 @@ module('Acceptance | pki workflow', function (hooks) {
     await click(SELECTORS.keysTab);
     assertEmptyState(assert, 'keys');
   });
-  test('shows pki beta banner to return to old pki on new pki configuration page', async function (assert) {
```
Comment (hashishaw) [inline ui/tests/acceptance/pki/pki-engine-workflow-test.js:76]: ✨

**Review by hashishaw — COMMENTED** [review]
Excellent work! Just a couple minor asks

File: ui/lib/pki/addon/components/page/pki-configuration-details.hbs [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+<InfoTableRow @label="Secret engine type" @value={{@mountConfig.engineType}} />
+<InfoTableRow @label="Path" @value={{@mountConfig.path}} />
+<InfoTableRow @label="Accessor" @value={{@mountConfig.accessor}} />
+<InfoTableRow @label="Local" @value={{@mountConfig.local}} />
+<InfoTableRow @label="Seal wrap" @value={{@mountConfig.sealWrap}} />
+<InfoTableRow @label="Default lease TTL" @value={{@mountConfig.config.defaultLeaseTtl}} />
+<InfoTableRow @label="Max lease TTL" @value={{@mountConfig.config.maxLeaseTtl}} />
+<InfoTableRow
+  @label="Allowed managed keys"
+  @value={{if @mountConfig.config.allowedManagedKeys @mountConfig.config.allowedManagedKeys "None"}}
```
[the remark this answers is not in the document]
Comment (kiannaquach) [reply ui/lib/pki/addon/components/page/pki-configuration-details.hbs]: great suggestion! i was wondering if there was a simpler way to do this!

**Review by hashishaw — APPROVED** [review]
Beautiful! :shipit:

**Pull request merged**